tensorrt_llm/_torch/attention_backend/sparse/utils.py

```
@@ -0,0 +1,39 @@
+from .rocket import (RocketKVCacheManager, RocketTrtllmAttention,
+                     RocketVanillaAttention)
+
+
+def get_sparse_attn_kv_cache_manager(
+        sparse_attn_config: "SparseAttentionConfig"):
```

Ruff (0.13.2)

Undefined name `SparseAttentionConfig`

(F821)